[2019-08-02 11:01:42,044] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: lm_cost_dag.cleaning_local_loop_interface_for_lm_cost 2019-08-02T00:00:00+00:00 [queued]>
[2019-08-02 11:01:42,049] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: lm_cost_dag.cleaning_local_loop_interface_for_lm_cost 2019-08-02T00:00:00+00:00 [queued]>
[2019-08-02 11:01:42,049] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-08-02 11:01:42,050] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-08-02 11:01:42,050] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-08-02 11:01:42,068] {__init__.py:1374} INFO - Executing <Task(PythonOperator): cleaning_local_loop_interface_for_lm_cost> on 2019-08-02T00:00:00+00:00
[2019-08-02 11:01:42,069] {base_task_runner.py:119} INFO - Running: ['airflow', 'run', 'lm_cost_dag', 'cleaning_local_loop_interface_for_lm_cost', '2019-08-02T00:00:00+00:00', '--job_id', '93', '--raw', '-sd', 'DAGS_FOLDER/lm_cost_func.py', '--cfg_path', '/tmp/tmpq09s682x']
[2019-08-02 11:01:42,559] {base_task_runner.py:101} INFO - Job 93: Subtask cleaning_local_loop_interface_for_lm_cost /home/aprao/.local/lib/python3.6/site-packages/airflow/configuration.py:590: DeprecationWarning: You have two airflow.cfg files: /home/aprao/airflow/airflow.cfg and /home/aprao/airflow_test/airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read /home/aprao/airflow_test/airflow.cfg, and you should remove the other file
[2019-08-02 11:01:42,559] {base_task_runner.py:101} INFO - Job 93: Subtask cleaning_local_loop_interface_for_lm_cost   category=DeprecationWarning,
[2019-08-02 11:01:42,866] {base_task_runner.py:101} INFO - Job 93: Subtask cleaning_local_loop_interface_for_lm_cost [2019-08-02 11:01:42,865] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-08-02 11:01:43,243] {base_task_runner.py:101} INFO - Job 93: Subtask cleaning_local_loop_interface_for_lm_cost [2019-08-02 11:01:43,243] {__init__.py:305} INFO - Filling up the DagBag from /home/aprao/airflow_test/dags/lm_cost_func.py
[2019-08-02 11:01:43,269] {base_task_runner.py:101} INFO - Job 93: Subtask cleaning_local_loop_interface_for_lm_cost [2019-08-02 11:01:43,269] {cli.py:517} INFO - Running <TaskInstance: lm_cost_dag.cleaning_local_loop_interface_for_lm_cost 2019-08-02T00:00:00+00:00 [running]> on host TCL60013867591.localdomain
[2019-08-02 11:01:43,277] {python_operator.py:104} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=lm_cost_dag
AIRFLOW_CTX_TASK_ID=cleaning_local_loop_interface_for_lm_cost
AIRFLOW_CTX_EXECUTION_DATE=2019-08-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=backfill_2019-08-02T00:00:00+00:00
[2019-08-02 11:01:43,278] {python_operator.py:113} INFO - Done. Returned value was: None
[2019-08-02 11:01:47,037] {logging_mixin.py:95} INFO - [2019-08-02 11:01:47,036] {jobs.py:195} ERROR - Scheduler heartbeat got an exception: (sqlite3.OperationalError) disk I/O error (Background on this error at: http://sqlalche.me/e/e3q8)
[2019-08-02 11:01:47,038] {logging_mixin.py:95} INFO - [2019-08-02 11:01:47,037] {jobs.py:2562} INFO - Task exited with return code 0
